### DISCUSSION TITLE
好好好

### DISCUSSION THREAD

**Discussion by stqfdyr** [XTLS/Xray-core#2984] [General]
之前一直被 `限速（姑且认为是这样吧）`的 reality 节点出狱了
之前修改过 `fingerprint` 为 `randomized`，但一直没啥用
放在那不管它了，今天想起来看看，突然出狱了，不限速了，直接拉满，好好好

**Comment by stqfdyr** [comment]
坏了，又进去了。。。。。。
:crying_cat_face::crying_cat_face::crying_cat_face:

**Comment by stqfdyr** [comment]
好了，又出来了 🙃

**Comment by stqfdyr** [comment]
坏了，又进去了。。。。。。
😿😿😿

**Comment by stqfdyr** [comment]
好了，又出来了 👍 🐱 🚀 🎸 💯

**Comment by stqfdyr** [comment]
坏了，成孟获了
😿😿😿

**Comment by qwerr0** [comment]
邻居不安分，同段IP会被重点监控，找个干净点的IP用Reality好像， 应该不会被限速
